docs/basics/rankine_cycle.rst

```diff
@@ -81,6 +81,39 @@ water return flow temperature.
 After rerunning, we will see that the condensation temperature and pressure
 are both automatically calculated by the specified terminal temperature value.
 
+Generating T-s Diagram
+^^^^^^^^^^^^^^^^^^^^^^
+To visualize the Rankine Cycle, we may generate a temperature (T) versus 
+entropy (s) diagram. To do this, we can import fluprodia (Fluid Property Diagram) 
+package. To successfully plot the T-s diagram, following steps are required:
+
+- Import the Package: Import fluprodia and create an object by passing the alias of the fluid.
+- Specify the Unit System: Set the unit system for all fluid properties.
+- Specify Custom Isolines: Define custom isolines for the diagram.
+- Calculate and Draw Isolines: Calculate and draw the isolines based on the model results.
+- Save and Export the Diagram: Save and export the completed T-s diagram.
+
+.. dropdown:: Click to expand to code section
+
+    .. literalinclude:: /../tutorial/basics/rankine.py
+        :language: python
+        :start-after: [sec_extra]
```

Comment:
So I understand this will affect the importing code of the current documentation, and to avoid changing the `sec_number`, I think it's ok to put a non-numeric `sec` here. But maybe we name it as `sec-Ts`
```suggestion
        :start-after: [sec_Ts]
```